crmsh/ui_resource.py

```diff
@@ -98,8 +98,8 @@ def set_deep_meta_attr_node(target_node, attr, value):
         for nvpair in nvpairs:
             nvpair.set("value", value)
     else:
-        for n in xmlutil.get_set_nodes(target_node, "meta_attributes", create=True):
-            xmlutil.set_attr(n, attr, value)
+        nodes = xmlutil.get_set_nodes(target_node, "meta_attributes", create=True)
+        xmlutil.set_attr(nodes[0], attr, value)
```

Comment:
Of course, purely from crmsh, we should avoid creation of multiple attributes sets first of all.

But given that already existing multiple attributes sets, the old way is probably the best we could do... Since the multiple attributes sets might have been intentionally configured by users together with different time-based rules, so the first set is not necessarily the one that applies to the expected time/need unless we could understand the contexts and the rules...

Reply:
Indeed. Okay.